api/pcve.py

```diff
@@ -1,17 +1,13 @@
 from flask import Flask
```

Comment:
Maybe you can think of using FastAPI instead of flask framework

Reply:
We thought of FastAPI., but current API requirements are very simple in nature.